src/main/resources/db/migration/V32__manuellbrevmottaker.sql

```diff
@@ -5,14 +5,16 @@ CREATE TABLE IF NOT EXISTS manuell_brevmottaker
     type           VARCHAR(50)                         NOT NULL,
     vergetype      VARCHAR(50),
     navn           VARCHAR                             NOT NULL,
-    adresselinje_1 VARCHAR                             NOT NULL,
+    ident          VARCHAR,
+    org_nr         VARCHAR,
+    adresselinje_1 VARCHAR,
     adresselinje_2 VARCHAR,
-    postnummer     VARCHAR                             NOT NULL,
-    poststed       VARCHAR                             NOT NULL,
-    landkode       VARCHAR(2)                          NOT NULL,
-    versjon        BIGINT       DEFAULT 0              NOT NULL,
-    opprettet_av   VARCHAR      DEFAULT 'VL'           NOT NULL,
-    opprettet_tid  TIMESTAMP(3) DEFAULT localtimestamp NOT NULL,
+    postnummer     VARCHAR,
+    poststed       VARCHAR,
+    landkode       VARCHAR(2),
+    versjon        BIGINT       DEFAULT 0,
+    opprettet_av   VARCHAR      DEFAULT 'VL',
+    opprettet_tid  TIMESTAMP(3) DEFAULT localtimestamp,
```

Comment:
Skulle disse fortsatt vært NOT NULL kanskje?
```suggestion
    versjon        BIGINT       DEFAULT 0                                     NOT NULL,
    opprettet_av   VARCHAR      DEFAULT 'VL'                        NOT NULL,
    opprettet_tid  TIMESTAMP(3) DEFAULT localtimestamp NOT NULL,
```